spark/src/main/scala/org/apache/spark/sql/delta/DeltaAnalysis.scala

```diff
@@ -362,6 +365,87 @@ class DeltaAnalysis(session: SparkSession)
             l.toString, "Unknown")
       }
 
+    // create uniform ingress table
+    case createUniformStmt: CreateUniformTableStatement =>
```

Comment:
The core logic for `CREATE` command, which utilizes the existing `SHALLOW CLONE` logic in `CloneTableCommand`.